docs/libs/ml/svm.md

```diff
@@ -186,5 +199,5 @@ svm.fit(trainingDS)
 val testingDS: DataSet[Vector] = env.readVectorFile(pathToTestingFile)
 
 // Calculate the predictions for the testing data set
-val predictionDS: DataSet[LabeledVector] = svm.predict(testingDS)
+val predictionDS: DataSet[(FlinkVector, Double)] = svm.predict(testingDS)
```

Comment:
I prefer FlinkVector in the library code at least to make sure we differentiate from Scala and Breeze. But you are right, for the example this import should suffice.